meta-xfstk/recipes-xfstk/xfstk/xfstk_1.8.1.bb

```diff
@@ -0,0 +1,20 @@
+DESCRIPTION = "Build XFSTK Downloader for recovery"
+LICENSE = "LGPL-2.1"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/LGPL-2.1;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+inherit cmake
```

Comment:
would we need BBCLASSEXTEND = "native"?

Reply:
That is a very good point, we could either have it as an extend or an inherit, for now I have added it as extend.